load-generator/src/main/java/com/amazon/opentelemetry/load/generator/emitter/StatsdMetricEmitter.java

```diff
@@ -0,0 +1,52 @@
+package com.amazon.opentelemetry.load.generator.emitter;
+
+import com.amazon.opentelemetry.load.generator.model.Parameter;
+
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+
+
+public class StatsdMetricEmitter extends MetricEmitter{
+
+    InetAddress ipAddress;
+    int portAddress;
+    private DatagramSocket socket;
+
+    public StatsdMetricEmitter(Parameter param) {
+        super();
+        this.param = param;
+    }
+
+    @Override
+    public void emitDataLoad() throws Exception {
+        this.setupProvider();
+        this.start(() -> nextDataPoint());
+    }
+
+    @Override
+    public void setupProvider() throws Exception {
+        ipAddress = InetAddress.getByName(param.getEndpoint().split(":",2)[0]);
+        portAddress = Integer.parseInt(param.getEndpoint().split(":",2)[1]);
+        socket = new DatagramSocket();
+    }
+
+    @Override
+    public void nextDataPoint() {
+        try {
+            sendstatsd();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void sendstatsd() throws IOException {
+        byte buf[];
+        String statsd = "statsdTestMetric1:1|g|#mykey:myvalue,mykey2:myvalue2";
+        buf = statsd.getBytes();
+        DatagramPacket packet = new DatagramPacket(buf, buf.length, ipAddress, portAddress);
+        socket.send(packet);
```

Comment:
did you check on cloudwatch console to ensure there're a mount of metrics appearing?

Reply:
Yes, checked.